buzzmobile/tests/unit/process/test_bearing.py

```diff
@@ -16,20 +16,16 @@ def test_distance(self):
         assert np.isclose(calculate_directions.get_distance(fix1, fix2), 2517000, rtol=.01)
 
 
-@with_roscore
-class TestBearingNode(unittest.TestCase):
-
+class TestBearingNode(RosTest):
     @with_launch_file('buzzmobile', 'test_params.launch')
     @launch_node('buzzmobile', 'bearing.py')
     def test_bearing_node(self):
-        with mock_pub('/fix', NavSatFix, queue_size=None) as fix_node:
+        with mock_pub('/fix', NavSatFix, queue_size=0) as fix_node:
             with check_topic('/buzzmobile/bearing', Float64) as ct:
                 # send mock data
                 fix_node.send(NavSatFix(None, None, 33.636700, -84.427863, None, None, None))
                 fix_node.send(NavSatFix(None, None, 39.029128, -111.838257, None, None, None))
 
                 # check the output from the node
-                await(ct.wait_for_message())
```

Comment:
see this:
https://stackoverflow.com/questions/1191374/using-module-subprocess-with-timeout